modules/serverless-blue-green-deploy.adoc

```diff
@@ -0,0 +1,115 @@
+[id="serverless-blue-green-deploy_{context}"]
+= Routing and managing traffic by using a blue-green deployment strategy
+
+You can safely reroute traffic from a live version of an app to a new version, by using a link:https://en.wikipedia.org/wiki/Blue-green_deployment[blue-green deployment strategy].
+
+.Procedure
+
+. Create and deploy an app as a Knative service.
+. Find the name of the first revision that was created when you deployed the service, by viewing the output from the following command:
++
+[source,terminal]
+----
+$ oc get configurations <service_name> -o=jsonpath='{.status.latestCreatedRevisionName}'
+----
+. Define a `Route` object to send inbound traffic to the revision:
++
+.Example Route
```

Comment:
Why are we using `Route` instead of `Service` here? I think we've been using Service everywhere else in the docs thus far?

Reply:
It came from the docs upstream: https://knative.dev/development/developer/serving/traffic-management/#routing-and-managing-traffic-with-bluegreen-deployment

If it needs to change we should update it in both places; can you open an issue for upstream docs if we change it here?
Also, how would "defining a service" send traffic to a revision? Wouldn't it _need_ to be a route, or would it be setting the same spec just on a service instead?

Reply:
Yes the same spec works inside the `traffic` section of the Service and that imo should be the default way of doing things (as everything we document downstream is based around `Service`)

Reply:
Updated to mention just adding the `traffic` section to the `spec` of the service.